src/Php84/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022-present Fabien Potencier
```

Comment:
```suggestion
Copyright (c) 2024-present Fabien Potencier
```
⌚

Reply:
Joys of copy-pasta 🥲. Thanks for the good catch.